pox/lib/addresses.py

```diff
@@ -690,7 +717,7 @@ def to_str (self, zero_drop = True, section_drop = True, ipv4 = None):
     by passing ipv4=True; this probably only makes sense if .is_ipv4_compatible
     (or .is_ipv4_mapped, of course).
     """
-    o = [ord(lo) | (ord(hi)<<8) for hi,lo in
+    o = [lo | (hi<<8) for hi,lo in
```

Comment:
I'm surprised this works.  I wonder if we should do something like I suggested for chr() and have an ordb() or whatever like `ordb = lambda x: struct.unpack('B', x)[0]`.

Reply:
The values is always bytes now, so string conversion with `ord` is not needed anymore